packages/core/constants/index.ts

```diff
@@ -106,6 +106,12 @@ export const RELATIVE_DATE_RANGES = [
         sort: undefined,
     },
 ];
+export const INITIAL_DATE_FILTER = [
```

Comment:
I think using this constant on line 47 would be ideal. Also this doesn't look like it needs to be an array